site/app/Resources/views/template/email_notification_template.txt.twig

```diff
@@ -0,0 +1,13 @@
+Hi {{ username }},
+
+You have a new notification from LibreCores.
+
+{{ content }}
+
+You are recieving this notification because you are subscribed
+to email notifications from LibreCores. You can unsubscribe from
+these notifications if you want to. For detailed preferences, go
+to {{ notificationSettingsUrl }} of your LibreCores account.
```

Comment:
- Please adjust this sentence as in the HTML version.
- Separate the footer with a "-- ".
- Wrap at 72 characters.